fail2ban/server/asyncserver.py

```diff
@@ -149,8 +149,11 @@ def start(self, sock, force):
 		self.__init = True
 		# TODO Add try..catch
 		# There's a bug report for Python 2.6/3.0 that use_poll=True yields some 2.5 incompatibilities:
-		logSys.debug("Detected Python 2.6 or greater. asyncore.loop() not using poll")
-		asyncore.loop(use_poll=False) # fixes the "Unexpected communication problem" issue on Python 2.6 and 3.0
+		if sys.version_info >= (2, 7) and sys.version_info < (2, 8): # if python 2.7 ...
+			logSys.debug("Detected Python 2.7. asyncore.loop() using poll")
+			asyncore.loop(use_poll=True) # workaround for the "Bad file descriptor" issue on Python 2.7, gh-161
```

Comment:
>    I understand that "unexpected communication problem" previously occurred
>    only in 2.6.

that is where it was detected... I don't remember if anyone was already
running 2.7 at that point.

>    Possibly it was already fixed in other versions?

who knows? ;) could be

>    Other workarround may be a try/catch around, because according to
>    issue10878 it occurred in asyncore by close. But imho it is not so
>    pretty...

I think I have tried something like that at some point but didn't
succeed to full solution